server/package.json

```diff
@@ -9,7 +9,7 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
+    "start": "set NODE_ENV=development&& nest start",
```

Comment:
Are you on Windows? I think it's "export" on unix no? 
What made you put it here vs .env, it didn't work there? (just curious, not saying it's best in .env!)

Reply:
yes, i am on windows. 
in this blog https://tacomanator.medium.com/environments-with-create-react-app-7b645312c09d.
i read that for react,

> **The value of NODE_ENV is set automatically to development (when using npm start), test (when using npm test) or production (when using npm build).**

So i thought of doing the same in nest.js(server), thats why i am setting the NODE_ENV to development for yarn start.